conda-recipe-apple/meta.yaml

```diff
@@ -27,8 +28,13 @@ requirements:
     - {{ pin_compatible('numpy') }}
     
 test:
+  requires:
+    - pytest
+    - scipy
   imports:
     - slycot
+  commands:
+    - pytest --pyargs slycot
```

Comment:
```suggestion
    - pytest
```

Should be taken care of by #110

Reply:
https://github.com/python-control/Slycot/blob/1bf49c1894745c62f7cbcd78f03052639ff10ef3/conda-recipe-apple/meta.yaml#L10

I cannot use the "suggest change" feature for this one, but the `_mkl_` infix needs to be changed here.

Reply:
The test does not seem to run from the source folder, conda arranges for a temporary folder for the test. `pytest` needs the `--pyargs slycot` here. As I understand it from the documentation, this effectively loads and searches the newly built `slycot` as installed in the test environment and runs its tests. 

This is also a nice way to ensure that an installed slycot is still intact.

Reply:
Ah, I also implemented the _mkl_ -> _obl_ change

Reply:
Re pytest: Sounds fair.

Why back to openblas and not apple system provided? Could this be merged with the openblas recipe dir again?

Reply:
You are right. I now got it to build & tests worked with Apple/Accelerate. Had to force the blas vendor for this with `-DBLA_VENDOR=Apple`.